libs/category/state/src/facades/category-facade.interface.ts

```diff
@@ -28,12 +30,11 @@ export interface DaffCategoryFacadeInterface<
   /**
    * The page configuration state for the selected category.
```

Comment:
I don't think the facade class should have docs at all.

Reply:
I think this is a good case for inheriting docs.

Reply:
I think we talked about this when we were writing docs a few months ago, and I thought we'd decided to upkeep both until we got dgeni to do the thing. I could be misremembering though

Reply:
I think this is already supported via jsdoc @lderrickable 

```
/** @inheritdoc */
```

Reply:
I would be pretty surprised if that just worked without needing to change our dgeni implementation at all.

Reply:
So I think there are three options:
1. get the @inheritdoc to work, which will probably require changes to our dgeni implementation (not recommended)
2. Make the docs match.
3. remove all documentation from the facade class until we get doc inheritance to work in another PR.

Because I don't think docs that don't match is an option.

Reply:
I elect for 1 because it's part of the jsdoc spec. I'd consider this a dgeni bug (or feature).

Reply:
See #1441

Reply:
45d7ca56